assets/queries/ansible/aws/ingress_not_restricted/metadata.json

```diff
@@ -0,0 +1,8 @@
+{
+  "id": "ingress_not_restricted",
+  "queryName": "Ingress Not Restricted",
+  "severity": "HIGH",
+  "category": "Network Ports Security",
+  "descriptionText": "AWS Security Group should restrict ingress access",
+  "descriptionUrl": "https://docs.ansible.com/ansible/latest/collections/amazon/aws/ec2_group_module.html"
+}
```

Comment:
[As a good POSIX practice](https://unix.stackexchange.com/a/18789) add end-of-file newline